services/jsdelivr/jsdelivr-base.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const Joi = require('joi')
+const { metric } = require('../../lib/text-formatters')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  total: Joi.number().required(),
+}).required()
+
+const keywords = ['jsDelivr', 'hits']
+
+const periodMap = {
+  hd: 'day',
+  hw: 'week',
+  hm: 'month',
+  hy: 'year',
+}
+
+class BaseJsDelivrService extends BaseJsonService {
+  static render({ period, hits }) {
+    return {
+      message: `${metric(hits)} hits/${periodMap[period]}`,
+      //color: downloadCount(hits),
+    }
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'jsDelivr',
+      color: 'orange',
```

Comment:
If we want to just make everything one colour, can we go with blue (which we use for "information" as opposed to good/bad/etc).

Reply:
I used orange as that was the color scheme used on their website and their badges.